### DISCUSSION TITLE
can I save what I have created on the file?

### DISCUSSION THREAD

**Discussion by robertwilli1** [pikasojs/pikaso#47] [Q&A] [answered]
can I save what I have created on the file, like if I created one circle and two rectangles and then I want to save it in my local drive, and then after a while I open the project and start creating objects again from where I left.

**Comment by raminious** [comment, marked as answer]
Hi @osamatan125 

Yes `editor.export.toJson()` gives you a json output that you can save that in a file or database. later, you can use `editor.import.json` to load that

https://pikaso.app/#/core/import-export

**Comment by robertwilli1** [comment]
Awesome, Thank you @raminious , I saved it in Local storage and it works perfectly.
